bqskit/runtime/base.py

```diff
@@ -82,7 +104,13 @@ def send_outgoing(node: ServerBase) -> None:
 
         outgoing[0].send((outgoing[1], outgoing[2]))
         node.logger.debug(f'Sent message {outgoing[1].name}.')
-        node.logger.log(1, f'{outgoing[2]}\n')
+
+        if outgoing[1] == RuntimeMessage.SUBMIT_BATCH:
```

Comment:
Perhaps also add a sample of the first and last element in the batch?

Reply:
Agreed, or some more information other than len should be logged